### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by mckillopjamie66** [community/community#75169] [Accessibility]
Account setup

**Comment by mckillopjamie66** [comment]
Please

**Comment by ktauchathuranga** [comment]
what exactly do you need help with?

**Comment by ghostinhershell** [comment]
Hi @mckillopjamie66, 

Thanks so much for reaching out. Can you provide more information on what you need help with?

You are more likely to get a useful response if you are explicit about what you need help with, giving a few more details might help someone give you a nudge in the right direction.

**Comment by ghostinhershell** [comment]
Hi @mckillopjamie66! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
